packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -1,3 +1,6 @@
+@import './clinical-lowlight-theme/Menu.module';
+@import './orion-fusion-theme/Menu.module';
```

Comment:
These should be imported from MenuItem.moudle.scss

Reply:
Done cf26d44